### ISSUE TITLE
Weird TV static effect on the in-game menu on Wario: MoD Title Screen

### ISSUE THREAD

**Issue by zurgeg** [DS-Homebrew/nds-bootstrap#1326]
**Your console:** DSi

**Launch Method:** TWLMenu++

**Version used:** 0.48.0

**Game tested/used/applicable:** Wario: Masters Of Disguise

**Expected Behavior:** No weird static effect

**Actual Behavior:** Weird static effect

### Steps to reproduce
1. Launch Wario MoD
2. Open the in-game menu

### Other notes
Nothing much.

**Comment by Epicpkmn11** [comment]
Does it look like these?
![](https://user-images.githubusercontent.com/41608708/127603230-1f4d90a3-da9f-480b-934b-70ef7cfa1d13.png) ![](https://user-images.githubusercontent.com/41608708/127603287-ef7c6b82-2853-4735-a9e9-76a1923d293c.png)

If so probably a duplicate of either #1278 and/or #1139.

**Comment by zurgeg** [comment]
> Does it look like these? ![](https://user-images.githubusercontent.com/41608708/127603230-1f4d90a3-da9f-480b-934b-70ef7cfa1d13.png) ![](https://user-images.githubusercontent.com/41608708/127603287-ef7c6b82-2853-4735-a9e9-76a1923d293c.png)
> 
> If so probably a duplicate of either #1278 and/or #1139.

No, it actually kinda looks like the effect on the TV screen in the Title Screen, which I think is the source of the issue

**Comment by zurgeg** [comment]
FYI, I think a better way to describe this is that it looks like an analouge TV stream "glitching" out or a VHS tape going bad, which can be observed in [this video](https://www.youtube.com/watch?v=q23Qw4poYjc). I can try to reproduce it on melonDS if needed.

**Comment by Kaisaan** [comment]
Tested this on my N3DS XL and everything seems fine to me. [Picture](https://cdn.discordapp.com/attachments/679100203969675295/944475340233932810/IMG_4720.jpg). Using Luma3DS 10.2.1, TWiLight Menu++ version 24.0.0, and nds-bootstrap 0.54.0. I think this can be closed now.

**Comment by RocketRobz** [comment]
> Tested this on my N3DS XL and everything seems fine to me. [Picture](https://cdn.discordapp.com/attachments/679100203969675295/944475340233932810/IMG_4720.jpg). Using Luma3DS 10.2.1, TWiLight Menu++ version 24.0.0, and nds-bootstrap 0.54.0. I think this can be closed now.

Try re-opening the in-game menu while making sure the top screen image is shown.

**Comment by Kaisaan** [comment]
I'm not able to get the top screen to show up whenever I open the in-game-menu. Additionally, the text for the menu is grey rather than white for other games. This still happens on TWiLight Menu++ version 24.1.0, and nds-bootstrap 0.54.1

**Comment by Radriant** [comment]
It seems to be somewhat mitigated, on a fresh boot up some black scan lines appear, varing from one pixel in height to 1/4 of the screen, they only stay from not more that 3-4 frames, weirdly enough after a reset I cant reproduce the problem

**Comment by spellboundtriangle** [comment]
I can confirm that it occurs when first booted, but it goes away after a while and does not return if you return to the title screen from the main menu or if you soft reset. It will happen again if you exit to TWLM++ and run the ROM again, though.

It also stops a bit earlier and only in 1 or 2 px y flashes instead of taking ~1/6 of the screen a couple times during it, when running at 133mhz.

Using 741378a

**Comment by spellboundtriangle** [comment]
Updated to latest as of writing nds-bootstrap and TWLM++ nightlies (dccb5c4). Results persist.

Interestingly, it _doesn't_ occur in B4DS. It doesn't occur with SCFG unlocked launched from flashcard, either (not sure about SCFG unlocked flashcard access on SD card, because TWLM++ just whitescreens upon trying to launch it).
